internal/devbox/secrets.go

```diff
@@ -18,7 +17,6 @@ func (d *Devbox) UninitializedSecrets(ctx context.Context) *envsec.Envsec {
 		},
 		IsDev:      build.IsDev,
 		Stderr:     d.stderr,
-		Store:      &jetstore.JetpackAPIStore{},
```

Comment:
is this change relevant to the PR?

Reply:
This is no longer used when we updated envsec dependency.